include/deal.II/distributed/shared_tria.h

```diff
@@ -72,6 +72,58 @@ namespace parallel
       typedef typename dealii::Triangulation<dim,spacedim>::cell_iterator        cell_iterator;
 
       /**
+       * Configuration flags for distributed Triangulations to be set in the
+       * constructor. Settings can be combined using bitwise OR.
+       */
+      enum Settings
+      {
+        /**
+         * Use METIS partitioner to partition active cells. This is the
+         * default partioning method.
+         */
+        partition_metis = 0x1,
+
+        /**
+         * Partition active cells with the same scheme used in the
+         * p4est library.
+         */
+        partition_zorder = 0x2,
+
+        /**
+         * Partition cells using a custom, user defined function. This is
+         * accomplished by connecting the post_refinement signal to the
+         * triangulation whenever it is first created and passing the user
+         * defined function through the signal using <code>std_cxx11::bind</code>.
+         * Here is an example:
+         *
+         *  @code
+         *     template <int dim>
+         *     void mypartition(parallel::shared::Triangulation<dim> &tria)
+         *     {
+         *       // user defined partitioning scheme
+         *     }
+         *
+         *     int main ()
+         *     {
+         *       parallel::shared::Triangulation<dim> tria(...,
+         *                                                 parallel::shared::Triangulation<dim>::Settings::partition_custom_signal);
+         *       shared_tria.signals.post_refinement.connect (std::bind(&mypartition<dim>, std::ref(shared_tria)));
+         *     }
+         *  @endcode
+         */
+        partition_custom_signal = 0x4,
+
+        /**
+         * This flags needs to be set to use the geometric multigrid
+         * functionality. This option requires additional computation and
+         * communication. Note: geometric multigrid is still a work in
+         * progress and not yet functional for shared triangulation.
```

Comment:
this is the sort of comment we will forget to remove when the MG actually starts working. I would just omit the note.